mitodl/open-discussions
# Bootcamps -> Courses

#### Pre-Flight checklist

#### What are the relevant tickets?
Closes #2792

#### What's this PR do?
Imports bootcamps as courses and removes lots of code that treated bootcamps as a distinct model

#### How should this be manually tested?
1. On master, run `manage.py backpopulate_bootcamp_data` and verify that on the course search page there is a 'Bootcamps' learning resource facet choice and 3 bootcamps returned when filtering by that.
2. Switch to this branch and restart the docker containers.  The course search page should no longer show the 'Bootcamps' type facet and otherwise everything should still work.
3.  Run `manage.py backpopulate_bootcamp_data` again.  Filter course search by the `Course` type and offered by `Bootcamps`.  There should be 3 results.  Clicking on them for details should work and show the correct info.
